hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/client/ScmClient.java

```diff
@@ -391,4 +391,35 @@ StatusAndMessages finalizeScmUpgrade(String upgradeClientID)
   StatusAndMessages queryUpgradeFinalizationProgress(
       String upgradeClientID, boolean force, boolean readonly)
       throws IOException;
+
+  List<HddsProtos.DatanodeDiskBalancerInfoProto> getDiskBalancerReport(
```

Comment:
Maybe we can call these two API both getDiskBalancerInfo?  Please also add comments that this count means top N DNs, not random DNs.

Reply:
@symious Could you explain a bit about the major difference between Report and Status?